sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2115,7 +2115,7 @@ object SQLConf {
     .createOptional
 
   val LEGACY_ARRAY_EXISTS_FOLLOWS_THREE_VALUED_LOGIC =
-    buildConf("spark.sql.legacy.arrayExistsFollowsThreeValuedLogic")
+    buildConf("spark.sql.legacy.arrayExistsFollowsThreeValuedLogic.enabled")
```

Comment:
ditto, would be better to make it a verb. e.g. `followThreeValuedLogicInArrayExists`